odoo/addons/base/models/ir_mail_server.py

```diff
@@ -184,6 +234,25 @@ class IrMail_Server(models.Model):
     smtp_debug = fields.Boolean(string='Debugging', help="If enabled, the full output of SMTP sessions will "
                                                          "be written to the server log at DEBUG level "
                                                          "(this is very verbose and may include confidential info!)")
+    smtp_ssl_trusted_public_keys = fields.Many2many(
+        'ir.attachment', string='Additional Trusted Public Keys',
```

Comment:
There is now the `certificate.{key,certificate}` models in the `certificate` module.

Reply:
meh... I need it in base...